tests/test_css.py

```diff
@@ -70,11 +76,19 @@ def test_allowed_css(data, styles, expected):
     p_double = "<p style='{0!s}'>bar</p>"
 
     if '"' in data:
+        if python2_and_unicode(data):
+            p_double = unicode(p_double)
         assert clean(p_double.format(data), styles=styles) == p_double.format(expected)
     else:
+        if python2_and_unicode(data):
+            p_single = unicode(p_single)
         assert clean(p_single.format(data), styles=styles) == p_single.format(expected)
 
 
+def python2_and_unicode(data):
+    return not isinstance(data, str)
```

Comment:
The function name suggests this should return `True` if it's running in Python 2 and this is unicode, but the test seems only to test if the data is `str`. I'm not sure what this is trying to do, but it seems like a mismatch.

Maybe something like this is better?:

```
def is_python2_str(data):
    return six.PY2 and isinstance(data, str)
```

Reply:
done in 33f294d.
Used the name `is_python2_unicode` which matches to the required logic.

Reply:
btw, I didn't know the `six` library. Thank you for suggestion!